src/Umbraco.Cms.ManagementApi/Controllers/Telemetry/GetTelemetryController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.ManagementApi.ViewModels.Telemetry;
+
+namespace Umbraco.Cms.ManagementApi.Controllers.Telemetry;
+
+public class GetTelemetryController : TelemetryControllerBase
+{
+    private readonly IMetricsConsentService _metricsConsentService;
+
+    public GetTelemetryController(IMetricsConsentService metricsConsentService) => _metricsConsentService = metricsConsentService;
+
+    [HttpGet("current")]
```

Comment:
```suggestion
    [HttpGet("level")]
```

IMO /telemetry/current does not tell what it is.. I prefer /telemetry/level.. Do you agree?

Reply:
Yep, will change it